package.json

```
@@ -16,7 +16,8 @@
     "test": "npm run test:spelling && npm run test:format && npm run test:build",
     "test:spelling": "cspell \"spec/**/*.md\" README.md",
     "format": "prettier --write \"**/*.{md,yml,yaml,json}\"",
-    "test:format": "prettier --check \"**/*.{md,yml,yaml,json}\" || echo \"\nTo resolve this, run: $(tput bold)npm run format$(tput sgr0)\" && exit 1",
+    "test:format": "prettier --check \"**/*.{md,yml,yaml,json}\" || npm run suggest:format",
+    "suggest:format": "echo \"\nTo resolve this, run: $(tput bold)npm run format$(tput sgr0)\" && exit 1",
```

I don't think anyone would ever want to run `suggest:format` so making it a top-level script doesn't seem desirable. Let's just add parens:

```suggestion
    "test:format": "prettier --check \"**/*.{md,yml,yaml,json}\" || ( echo \"\nTo resolve this, run: $(tput bold)npm run format$(tput sgr0)\" && exit 1 )",
```

```bash
$ (true || ( echo 'hi' && exit 1 ))
$ echo $?
0
$ (false || ( echo 'hi' && exit 1 ))
hi
$ echo $?
1
```

---

Actually, we don't use `yarn` so this is unsafe. Windows shell uses different syntax IIRC, and though yarn tries to make it consistent, npm doesn't.